indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -195,4 +195,27 @@
             $(this).closest('tr').toggleClass('selected', this.checked);
         }).trigger('change');
     };
+
+    global.showThemeSettingsForm = function showThemeSettingsForm(formUrl, lastFieldId) {
+        $('#timetable_theme').on('change', function() {
+            $.ajax({
+                url: formUrl,
+                data: {
+                    theme: $(this).val()
+                },
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function(data) {
+                    var lastField = $(lastFieldId);
```

Comment:
If the argument is called `lastFieldId` it shouldn't contain the `#` but it should rather be added here